src/geo/projection/globe.js

```diff
@@ -258,7 +258,7 @@ export function calculateGlobeMercatorMatrix(tr: Transform): mat4 {
     mat4.translate(posMatrix, posMatrix, [point.x, point.y, 0.0]);
     mat4.scale(posMatrix, posMatrix, [ws, ws, zScale]);
 
-    return posMatrix;
+    return Float32Array.from(posMatrix);
```

Comment:
It does explicitly convert something which was previously `Float64Array` into `Float32Array`, because it's later passed to WebGL `uniformMatrix`. This instance doesn't seem to hurt precision but please double check if you're in doubt.

Reply:
Yes, that's right!